datasets/GM0340_rhenen/GM0340_rhenen.derived.ad

```diff
@@ -136,43 +136,43 @@
 - solar_pv_profile_2_share = 0.0
 - annual_infrastructure_cost_gas = 1195421.857
 - areable_land = 21.97
-- co2_emission_1990 = 0.151582735
+- co2_emission_1990 = 0.147750174637452
```

Comment:
This seems like a generic update that applies regardless of the analysis_year. Are you sure that it should be reverted?

Reply:
This has not been changed by us but due to more recent figures of the data-source. I'll check why and how this changes..